### ISSUE TITLE
Disabling random crop when augmenting

### ISSUE THREAD

**Issue by tannisroot** [lucidrains/stylegan2-pytorch#119]
Hi, I'm training on a small handpicked and manually cropped dataset (~450) with some of the parameters people here shared (including augmentation set to 0.3) and while the results it produces after 15 epochs are quite nice, a lot of the results seem to be cropped in quite undesirable ways, which i assume is due to augmentation option doing random cropping on the input images. I wish I could show what I'm talking about exactly, but it's a bit too embarrassing to share xD
So I was wondering, can it be possible to keep it augmenting the data in all the ways possible but omit random cropping?

**Comment by tannisroot** [comment]
Hey, sorry, I haven't been able to test this change yet because something regressed between 18.3 and 20.1 that causes stylegan2 to produce garbage rainbow on my small dataset with these settings (--aug-prob 0.3 --dataset-aug-prob 0.6 --attn-layers [1,2] --save_every 100 --image-size 128 --batch-size 5 --gradient-accumulate-every 6 --cl-reg --trunc-psi 0.9 ) and I haven't had the time to bisect the regression.
I will try to backport the augmentation change on top of a working working and get back to you, though I think it would be nice if it was an option so that people could experiment with different values for their datasets.

**Comment by tannisroot** [comment]
Update - I had very good results by changing crop values in 0.18.6 to `0.85, 0.95`! However, this might be too low for some datasets, so making it an option might be a good idea after all.

**Comment by tannisroot** [comment]
removing cutout augmentation with `--aug-types` should suit my purposes
